modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+    exporters:
+      jaeger:
+        endpoint: jaeger-production-collector-headless.tracing-system.svc:14250
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+      prometheus:
+        endpoint: 0.0.0.0:8889
+        resource_to_telemetry_conversion:
+          enabled: true # by default resource attributes are dropped
+    service: <1>
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: []
+          exporters: [jaeger]
+        metrics:
+          receivers: [otlp]
+          processors: []
+          exporters: [prometheus]
+----
+<1> If a component is configured but not defined in the `service` section, the component is not enabled.
+====
+
+.Parameters used by the Operator to define the OpenTelemetry Collector
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default
+|receivers:
+|A receiver is how data gets into the Collector. By default, no receivers are configured. There must be at least one enabled receiver for a configuration to be considered valid. Receivers are enabled by being added to a pipeline.
+|`otlp`, `jaeger`, `zipkin`
+|None
+
+|processors:
+|Processors run through the data between it is received and exported. By default, no processors are enabled.
+|
+|None
+
+|exporters:
+|An exporter sends data to one or more back ends or destinations. By default, no exporters are configured. There must be at least one enabled exporter for a configuration to be considered valid. Exporters are enabled by being added to a pipeline. Exporters might be used with their default settings, but many require configuration to specify at least the destination and security settings.
+|`otlp`, `otlphttp`, `jaeger`, `logging`, `prometheus`
+|None
+
+|service:
+  pipelines:
+|Components are enabled by adding them to a pipeline under `services.pipeline`.
+|
+|
+
+|service:
+  pipelines:
+    traces:
+      receivers:
+|You enable receivers for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    traces:
+      processors:
+|You enable processors for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    traces:
+      exporters:
+|You enable exporters for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      receivers:
+|You enable receivers for metrics by adding them under `service.pipelines.metrics`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      processors:
+|You enable processors for metircs by adding them under `service.pipelines.metrics`.
+|
+|None
+
+|service:
+  pipelines:
+    metrics:
+      exporters:
+|You enable exporters for metrics by adding them under `service.pipelines.metrics`.
+|
+|None
+|===
+
+== OpenTelemetry Collector components
+
+=== Receivers
+
+==== OTLP Receiver
```

Comment:
[Anchoring in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)
Missing anchor IDs

Reply:
Thank you for the link. Added.